http/1.5/modules/ROOT/pages/http-troubleshooting.adoc

```diff
@@ -24,29 +30,28 @@ To enable HTTP wire logging, the following entry should be added to your artifac
 </Configuration>
 ----
 
-== TLS Debug
+[start=6]
+. Save your changes.
+. Click the project name in *Package Explorer* and then click *Run* > *Run As* > *Mule Application*.
 
-When SSL failures are seen or suspected because HTTPS is being used, usually connections cannot even be established and wire logging is not helpful.
-In this situations, SSL debug should be enabled which will log all SSL related data, such as connection attempts, handshakes, negotiations and so on.
+== Enable TLS Debugging
 
-Logs can then be examined to determine whether the correct certificates are in place, a cipher or protocol mismatch is occurring or why connectivity is failing.
+When SSL failures occur because of HTTPS, usually connections cannot be established and wire logging is not helpful. In these situations, enable TLS debugging that logs all SSL related data, such as connection attempts, handshakes and negotiations.
 
-To enable TLS debugging, the following system property needs to be configured: `-Djavax.net.debug=ssl`.
+You can then examine the logs to determine whether the correct certificates are in place, why a cipher or protocol mismatch occurs or why connectivity fails.
 
-== Common Issues
+To enable TLS debugging, configure the following system property: `-Djavax.net.debug=ssl`.
 
-These are some of the common issues seen while working with HTTP.
 
-=== Host Header Misuse
+== Troubleshoot Host Header Misuse
 
-The `Host` header is used by HTTP clients to indicate the host and port number of the resource being requested. It is a mandatory header which the HTTP request
-autogenerates considering the configured host and port.
+HTTP clients use the `Host` header to indicate the host and port number of the resource requested. The HTTP Connector *Request* operation autogenerates this mandatory header considering the configured host and port.
 
 Often, servers do not account for the port information because it's optional, and fail when receiving a request that has both the host and port data.
 
-Such failures should be reported to the server owners. While a fix is provided, the HTTP request operation can be modified to workaround the issue by providing an explicit `Host` header without the port section.
+Such failures should be reported to the server owners. As a workaround for the issue, modify the *Request* operation by providing an explicit `Host` header without the port section.
```

Comment:
Alternative for minimalism?

The HTTP Connector *Request* operation automatically generates a `Host` header that indicates both the host and the port number of a resource requested from a server. However, servers often consider the port information optional and fail if they receive a request containing it.

Report such failures to the server owner to correct. Meanwhile, work around the issue by manually modifying the *Request* operation to output a `Host` header without the port information.